docs/api/structures/web-preferences.md

```diff
@@ -10,6 +10,8 @@
   enabling Node.js support in sub-frames such as iframes and child windows. All your preloads will load for
   every iframe, you can use `process.isMainFrame` to determine if you are
   in the main frame or not.
+* `frozenIntrinsics` boolean (optional) - Experimental option for passing
+  [`--frozen-intrinsics`](https://nodejs.org/api/cli.html#--frozen-intrinsics) to Node.js.
```

Comment:
The document should clarity whether this requires `nodeIntegration: true` to work, and how it works with `contextIsolate: true`.